src/components/CountDown.tsx

```diff
@@ -9,8 +9,8 @@ import { useTheme, DarkTheme } from '@react-navigation/native';
 dayjs.extend(duration);
 dayjs.extend(isSameOrBefore);
 
-const start = dayjs(new Date(2021, 11, 10, 16, 30, 0)),
-  end = dayjs(new Date(2021, 11, 12, 16, 30, 0)),
+const start = dayjs(new Date(2021, 10, 12, 21, 0, 0)),
```

Comment:
The reason this is a nitpick is because we're not likely to change the hackathon date and time, but I wanted to point this out because it's good to know.

Reply:
We could place it as an iso string in .env but since its not likely to change I'll pass it to the next team.